vms/platformvm/blocks/executor/verifier.go

```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2022-2023, Chain4Travel AG. All rights reserved.
```

Comment:
I suggest to use our own verifier (instance in manager) instead of these invasive changes